networking/request-headers.html.md

```diff
@@ -9,7 +9,7 @@ Request headers carry information that is specific to the incoming request and i
 ## Request Headers
 
 ### `Fly-Client-IP`
-**Client IP Address**: The IP address the Fly Proxy accepted a connection from. This will be the client making the initial request and as such, will also appear at the start of the `X-Forwarded-For` addresses. 
+**Client IP Address**: The IP address the Fly Proxy accepted a connection from. You can use `Fly-Client-IP` to get the IP address of the client making the initial request if you aren't using any other proxy service for your app in front the Fly.io platform. When you use a proxy, such as Cloudflare, requests will appear to come from the proxy service's IP addresses, rather than the client making the request. In that case, you'll need to parse the IP addresses in the [`X-Forwarded-For` header](#x-forwarded-for) to find the client IP address.
```

Comment:
I'm not sure this is accurate... is this also the case if the user accessing the app is spoofing their own IP?

Reply:
@andie787 A parser for `X-Forwarded-For` as prescribed by the MDN doc shouldn't be fooled by a user spoofing their own IP based on that header, because it should stop at any point in that header where it sees an untrusted IP.

I'm wondering whether something like this would work better:

> **Client IP Address**: The IP address of the client _from the perspective of Fly Proxy_. If you are using another reverse proxy service in front of the Fly.io platform, this header will be set to the proxy service's IP address, rather than the client making the request. In that case, you'll need to parse the IP addresses in the [`X-Forwarded-For` header](#x-forwarded-for) to find the client IP address.